Output-file rewriting removes '.py' in too many places

The last line of  `pytest_mypy_plugins.items.replace_fpath_with_module_name()` does a blanket removal of all ".py" strings:

    return line.strip().replace(".py", "")

This destroys paths in log information when `mypy.verbosity > 0`. Paths like `"$HOME/.pyenv"` and `".../python3.6/.../__init__.pyi"` are converted into random strings.

I suspect you are only trying to replace `"main.py: 1: note: ..."` -type strings at the beginning of the line. If that is true (and I really haven't explored enough to be sure, so please check this) then a simple replacement my be `replace(".py:", ":")` to ensure the offending .py looks like a filespec. 

Otherwise maybe only do the replacement in the first colon-delimited field?

When I have these settings in a test .yaml file (note the `verbosity`):
```
  mypy_config: |
    [mypy]
    # !! IMPORTANT !!
    namespace_packages = True
    verbosity = 1
```

This line appears in my output:

    E     LOG:  Metadata fresh for genericpath: file /home/aghast/env/versions/3.6.9/envs/venv-aghast-utils/lib/python3.6/site-packages/mypy/typeshed/stdlib/2and3/genericpathi (diff)

I don't have a `env` directory under $HOME. But I do have a `.pyenv` directory. And I don't have a file named `.../2and3/genericpathi` but I do have one named `.../2and3/genericpath.pyi`.

The difference in both cases is that the `replace(".py", "")` is deleting too much.

--- a/.travis.yml
+++ b/.travis.yml
@@ -34,7 +34,7 @@ jobs:
     script: 'black --check pytest_mypy_plugins setup.py'
   - name: Lint with isort
     python: 3.7
-    script: 'isort --check --diff'
+    script: 'isort --check --diff .'
 
 before_install: |
   # Upgrade pip, setuptools, and wheel

--- a/pytest_mypy_plugins/collect.py
+++ b/pytest_mypy_plugins/collect.py
@@ -2,14 +2,14 @@ import os
 import platform
 import sys
 import tempfile
-from typing import TYPE_CHECKING, Any, Dict, Iterator, List, Optional, Mapping
+from typing import TYPE_CHECKING, Any, Dict, Iterator, List, Mapping, Optional
 
+import pystache
 import pytest
 import yaml
 from _pytest.config.argparsing import Parser
 from _pytest.nodes import Node
 from py._path.local import LocalPath
-import pystache
 
 from pytest_mypy_plugins import utils
 

--- a/pytest_mypy_plugins/item.py
+++ b/pytest_mypy_plugins/item.py
@@ -53,7 +53,7 @@ def replace_fpath_with_module_name(line: str, rootdir: Path) -> str:
         return line
     out_fpath, res_line = line.split(":", 1)
     line = os.path.relpath(out_fpath, start=rootdir) + ":" + res_line
-    return line.strip().replace(".py", "")
+    return line.strip().replace(".py:", ":")
 
 
 class ReturnCodes:

--- a/setup.cfg
+++ b/setup.cfg
@@ -1,4 +1,3 @@
 [isort]
 include_trailing_comma = true
 multi_line_output = 3
-wrap_length = 120

